gslib/commands/defacl.py

```diff
@@ -191,6 +193,38 @@ class DefAclCommand(Command):
       },
   )
 
+
+  def get_gcloud_storage_args(self):
+    sub_command = self.args.pop(0)
+    if sub_command == 'get':
+      gcloud_storage_map = GcloudStorageMap(
+          gcloud_command=[
+            'storage', 'buckets', 'describe',
+            '--format=multi(defaultObjectAcl:format=json)'
+        ],
+          flag_map={},
+        )
+
+    elif sub_command == 'set':
+      if (os.path.isfile(self.args[0])):
+        gcloud_storage_map = GcloudStorageMap(
+          gcloud_command=[
+            'storage', 'buckets', 'update',
+            '--default-object-acl-file=' + self.args.pop(0)
+          ],
+          flag_map={},
+        )
+      else:
+        gcloud_storage_map = GcloudStorageMap(
+          gcloud_command=[
+            'storage', 'buckets', 'update',
+            '--predefined-default-object-acl=' + self.args.pop(0)
```

Comment:
This won't translate XML predefined ACL strings to JSON ones for gcloud storage

Reply:
Added xml to json translation helper